redhat-cop/ah_configuration
# Make galaxy.yml persistent and move out ansible.cfg

# What does this PR do?
Makes galaxy.yml persistent and moves the template out of root. Also moves ansible.cfg from root.

This makes it installable from source


# How should this be tested?
1. Test we can install from source
2. We will need to perform a release and ensure that the galaxy.yml remains in the repo with `-devel` appended to the version that's just been released. We need to also be sure that the galaxy.yml of the released version is correct
# Is there a relevant Issue open for this?


resolves #280 

# Other Relevant info, PRs, etc
Required https://github.com/redhat-cop/ansible_collections_tooling/pull/33 to be merged
